### PR TITLE
Merge release/2.1.3xx

### PR DESCRIPTION
**Pull request by livarcocc**
This was a clean merge.

Just reverted the change to BranchInfo.props.

### CODE REVIEW

**Comment by livarcocc** [comment]
cc @mikeharder

File: test/dotnet-new.Tests/NuGet.tempaspnetpatch.config [hunk 1]
```diff
@@ -6,5 +6,6 @@
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="DotnetCore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
+    <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
```
Comment (johnbeisner) [inline test/dotnet-new.Tests/NuGet.tempaspnetpatch.config]: This was a fix for specific ASP.NETcore.App versions in the orchestrated build for release/2.1.3xx that took dependencies on non-published packages: System.IO.Pipelines 0.1.0-preview1-180129-1 and System.Text.Encodings.Web.Utf8 0.1.0-preview1-180129-1 - This change may not be necessary in CLI:master.

File: build/NugetConfigFile.targets [hunk 2]
```diff
@@ -28,9 +28,8 @@
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="vstest" value="https://dotnet.myget.org/F/vstest/api/v3/index.json" />
-    <add key="web-api" value="https://dotnet.myget.org/F/dotnet-web/api/v3/index.json" />
-    <add key="symreader-native" value="https://dotnet.myget.org/F/symreader-native/api/v3/index.json" />
-    <add key="nuget-build" value="https://dotnet.myget.org/F/nuget-build/api/v3/index.json" /> 
+    <add key="nuget-build" value="https://dotnet.myget.org/F/nuget-build/api/v3/index.json" />
+    <add key="dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
```
Comment (johnbeisner) [inline build/NugetConfigFile.targets]: The addition of the 'dotnet-corefxlab' feed was a fix for specific ASP.NETcore.App versions in the orchestrated build for release/2.1.3xx that took dependencies on non-published packages: System.IO.Pipelines 0.1.0-preview1-180129-1 and System.Text.Encodings.Web.Utf8 0.1.0-preview1-180129-1 - That specific line may not be necessary in CLI:master.

File: test/dotnet-new.Tests/NuGet.tempaspnetpatch.config (hunk 1 above)
Comment (livarcocc) [reply test/dotnet-new.Tests/NuGet.tempaspnetpatch.config]: If this is not necessary, we should remove it in 2.1.3xx and once that branch merges back into master, master will be fixed.

Comment (johnbeisner) [reply test/dotnet-new.Tests/NuGet.tempaspnetpatch.config]: It is indeed necessary in release/2.1.3xx at this time - we can merge the addition of the line and then merge the removal as you say.

**Pull request merged**